package.json

```diff
@@ -56,7 +56,7 @@
     "@types/meow": "^4.0.0",
     "@types/ncp": "^2.0.0",
     "@types/node": "^9.3.0",
-    "@types/pify": "3.0.0",
+    "@types/pify": "3.0.1",
```

Comment:
Folks, any reason for pinning the version here?

Reply:
I can't remember a reason for this. Looking at the history, this format has been around since the initial version, so I am pretty sure the pinning was unintentional.